lib/env.js

```diff
@@ -31,7 +31,8 @@ let environmentConfig = {
   BADGES_HOST: str({default: 'badges.greenkeeper.io'}),
   NPMHOOKS_SECRET: str({ devDefault: 'make-secrets-great-again' }),
   IS_ENTERPRISE: bool({ default: false }),
-  HOOKS_HOST: str({default: 'hooks.greenkeeper.io'})
+  HOOKS_HOST: str({default: 'hooks.greenkeeper.io'}),
+  GK_HOST: host({default: 'greenkeeper.io'})
```

Comment:
FYI: The reason why I didn't go with `GK_HOST` but with separate env vars was because we might want to get rid of the subdomains. Nothing wrong with your approach, just wanted to give some insight why I chose the other way in case it wasn't clear.

Reply:
@rmehner not sure how the subdomains are related? Yes we might wanna get rid of them, but then I still need the URL to the Greenkeeper instance itself in [certain cases](https://github.com/greenkeeperio/greenkeeper/pull/763/commits/29b23321895dfd4c0314fc7f5ea88dc3e4feb8bc#diff-b2cebe45e487360c83b2030ee57ec561L165).

Reply:
Ha, read the commit wrong, but also just wanted to give a heads up to make things a bit clearer. Turns out it was even more confusing. Yay me :)

Reply:
Is `host` a valid type here? `GK_HOST: host({default: 'greenkeeper.io'})` Shouldn’t it be `str` like in the line above?